### ISSUE TITLE
Is this implement same as the Keras implementation with Triplet loss?

### ISSUE THREAD

**Issue by smith-co** [anukash/Face_recognition_siamese_network#2]
Keras has a duplicate image similarity detection with triplet loss [here](https://keras.io/examples/vision/siamese_network/).

Is this example you have here is same as the Keras example? If not how are these examples differ?

Would the implementation in this repo work better than the Keras example?

**Comment by smith-co** [comment]
@anukash would be interested to know how this implementation is different from Keras example of duplicate image similarity detection with triplet loss?
